README.md

```diff
@@ -8,14 +10,18 @@ It currently does not yet implement many important features, such as SOCKS,
 HTTP/2, and many of the predefined JS functions that PAC files use.
 Contributions are welcome, please reach out to me if you'd like to help!
 
-## Installation and Usage
+## Download
+
+Releases can be downloaded from [here][1].
```

Comment:
Using the string "here" as an anchor for a link is poor. Better to describe what is "here":

Releases can be downloaded from the [alpaca github releases page][1].

That way the link text describes the link, which can also be useful for accessibility.